### PR TITLE
Feature - align Serilog approach with Console logging

### PR DESCRIPTION
**Pull request by stijnmoreels**
Closes https://github.com/arcus-azure/arcus.templates/issues/127
Closes https://github.com/arcus-azure/arcus.templates/issues/114

### CODE REVIEW

File: src/Arcus.Templates.ServiceBus.Queue/Arcus.Templates.ServiceBus.Queue.csproj [hunk 1]
```diff
@@ -55,7 +55,7 @@
     <PackageReference Include="Serilog" Version="2.9.0" Condition="'$(ExcludeSerilog)' == 'false'" />
     <PackageReference Include="Serilog.Extensions.Hosting" Version="3.0.0" Condition="'$(ExcludeSerilog)' == 'false'" />
     <PackageReference Include="Serilog.Sinks.ApplicationInsights" Version="3.1.0" Condition="'$(ExcludeSerilog)' == 'false'" />
-    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.9.10" />
+    <PackageReference Include="Serilog.Sinks.Console" Version="3.1.1" Condition="'$(Configuration)' == 'Debug'"/>
```
Comment (stijnmoreels) [inline src/Arcus.Templates.ServiceBus.Queue/Arcus.Templates.ServiceBus.Queue.csproj]: It's not a removal, apperently, the Container tools package was duplicated here.

File: src/Arcus.Templates.ServiceBus.Topic/Arcus.Templates.ServiceBus.Topic.csproj [hunk 2]
```diff
@@ -56,7 +56,6 @@
     <PackageReference Include="Serilog.Extensions.Hosting" Version="3.0.0" Condition="'$(ExcludeSerilog)' == 'false'" />
     <PackageReference Include="Serilog.Sinks.ApplicationInsights" Version="3.1.0" Condition="'$(ExcludeSerilog)' == 'false'" />
     <PackageReference Include="Serilog.Sinks.Console" Version="3.1.1" Condition="'$(Configuration)' == 'Debug'" />
-    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.9.10" />
```
Comment (stijnmoreels) [inline src/Arcus.Templates.ServiceBus.Topic/Arcus.Templates.ServiceBus.Topic.csproj]: It's not a removal, apperently, the Container tools package was duplicated here.

File: src/Arcus.Templates.ServiceBus.Topic/Program.cs [hunk 3]
```diff
@@ -83,5 +78,26 @@ public static IHostBuilder CreateHostBuilder(string[] args)
...
+            LoggerConfiguration currentLoggerConfiguration)
+        {
+            var instrumentationKey = hostContext.Configuration.GetValue<string>(ApplicationInsightsInstrumentationKeyName);
+
+            currentLoggerConfiguration
+                .MinimumLevel.Debug()
+                .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
+                .Enrich.FromLogContext()
+//#if DEBUG
+#if DEBUG
```
Comment (tomkerkhove) [inline src/Arcus.Templates.ServiceBus.Topic/Program.cs]: See issue, we should always add Console

File: src/Arcus.Templates.ServiceBus.Queue/Program.cs [hunk 4]
```diff
@@ -83,5 +78,26 @@ public static IHostBuilder CreateHostBuilder(string[] args)
...
+            LoggerConfiguration currentLoggerConfiguration)
+        {
+            var instrumentationKey = hostContext.Configuration.GetValue<string>(ApplicationInsightsInstrumentationKeyName);
+
+            currentLoggerConfiguration
+                .MinimumLevel.Debug()
+                .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
+                .Enrich.FromLogContext()
+//#if DEBUG
+#if DEBUG
```
Comment (tomkerkhove) [inline src/Arcus.Templates.ServiceBus.Queue/Program.cs]: See issue, we should always add Console

File: src/Arcus.Templates.ServiceBus.Topic/Program.cs (hunk 3 above)
Comment (stijnmoreels) [reply src/Arcus.Templates.ServiceBus.Topic/Program.cs]: 👍

File: src/Arcus.Templates.ServiceBus.Queue/Program.cs (hunk 4 above)
Comment (stijnmoreels) [reply src/Arcus.Templates.ServiceBus.Queue/Program.cs]: 👍

**Pull request merged**